File: tests/fixtures/java-simple/src/utils/Logger.java
package utils;

/** Simple logging utility. */
public class Logger {
    private final String context;

    public Logger(String context) {
        this.context = context;
    }

    public void info(String message) {
        System.out.println("[INFO] " + context + ": " + message);
    }

    public void error(String message) {
        System.err.println("[ERROR] " + context + ": " + message);
    }

    public void debug(String message) {
        System.out.println("[DEBUG] " + context + ": " + message);
    }
}

File: tests/fixtures/kotlin-simple/src/utils/Logger.kt
package utils

/** Simple logging utility. */
class Logger(private val context: String) {
    fun info(message: String) {
        println("[INFO] $context: $message")
    }

    fun error(message: String) {
        System.err.println("[ERROR] $context: $message")
    }

    fun debug(message: String) {
        println("[DEBUG] $context: $message")
    }
}
